src/main/java/at/hannibal2/skyhanni/config/features/garden/CropStartLocationConfig.java

```diff
@@ -3,14 +3,38 @@
 import at.hannibal2.skyhanni.config.FeatureToggle;
 import com.google.gson.annotations.Expose;
 import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
 import io.github.moulberry.moulconfig.annotations.ConfigOption;
 
 public class CropStartLocationConfig {
 
     @Expose
-    @ConfigOption(name = "Enable", desc = "Show the start waypoint for the farm of your current tool in hand. Do §e/shcropstartlocation §7to change the waypoint again.")
+    @ConfigOption(name = "Enable", desc = "Show waypoints for the farm of your current tool in hand. ")
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean enabled = false;
 
+    @Expose
+    @ConfigOption(name = "Crop Location Mode", desc = "Whether to show waypoint at start location (set with §e/shcropstartlocation §7) or last farmed location.")
+    @ConfigEditorDropdown
+    public CropLocationMode cropLocationMode = CropLocationMode.START;
```

Comment:
rename the field to `mode`, as the full name is redundant.

Reply:
Sorry for the delay. I fixed the field name. Moving the waypoint label down still caused problems when very close and looked awkward when it was too far offset, so I changed the waypoint color to light purple (which should contrast with almost all farm designs and with the other waypoint label colors).

![2024-05-16_10 20 21](https://github.com/hannibal002/SkyHanni/assets/16139460/e049d3e3-2344-44f7-b122-1ce0d7941405)

I also rounded the crop start location to the nearest block for consistency since the last farmed waypoint is block-aligned.